orchestrator/db/sorting/process.py

```diff
@@ -14,7 +14,11 @@
 
 def generic_process_relation_sort(field: Column) -> Callable[[QueryType, SortOrder], QueryType]:
     def sort_function(query: QueryType, order: SortOrder) -> QueryType:
-        joined_query = query.join(ProcessSubscriptionTable).join(SubscriptionTable).join(ProductTable)
+        if isinstance(query, Select):
+            joined_query: QueryType = query.join(ProcessSubscriptionTable).join(SubscriptionTable).join(ProductTable)
+        else:
+            # Can't join on CompoundSelect
```

Comment:
Would it make sense to log a warning in this case?

Reply:
No, it's more an explanation why we need to check for the instance type